lib/service_registry.rb

```diff
@@ -0,0 +1,17 @@
+class ServiceRegistry
```

Comment:
Yeah, this is a result of me trying out different things. I agree it's a good idea, but I'm not sure how to achieve it. Cause it would be C&P, so it'll get out of sync anyway.